README.md

```diff
@@ -337,26 +337,35 @@ The following composer scripts are aliases for running these tools.
 * Check for PHP lint errors
   `./bin/composer php-lint`
 
+## Checking Code with PHPStan
+
+[PHPStan](https://phpstan.org/) is available to statically analyze custom theme and module code for correctness.
+
+It should be installed automatically on a local environment via `composer install`.
```

Comment:
I'd like to tweak this wording, because people understandably get confused and run `composer install` on their machine instead of in the composer container. 

How about: 
> It's defined as a dev dependency in composer.json and will be installed automatically when you run the build scripts or `bin/composer install`.

Reply:
done